### PR TITLE
[MozillaCACerts] Update certs

### PR DESCRIPTION
**Pull request by jeremiahpslewis**

### CODE REVIEW

File: M/MozillaCACerts/build_tarballs.jl [hunk 1]
```diff
@@ -37,4 +37,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat = "1.6")
```
Comment (giordano) [inline M/MozillaCACerts/build_tarballs.jl:40]: > build for experimental plat compat

What?!?

Comment (jeremiahpslewis) [reply M/MozillaCACerts/build_tarballs.jl:40]: Isn't a julia_compat argument necessary for usage by apple silicon?

(Experimental platform compatibility)

Comment (giordano) [reply M/MozillaCACerts/build_tarballs.jl:40]: But this is a platform-independent package, you aren't even "building" anything

Comment (jeremiahpslewis) [reply M/MozillaCACerts/build_tarballs.jl:40]: Ok, so compat is unnecessary? But reasonable to update to latest certs?

Comment (giordano) [reply M/MozillaCACerts/build_tarballs.jl:40]: > Ok, so compat is unnecessary?

Yes

> But reasonable to update to latest certs?

Not sure how they (updating the certificates and changing the compat bound) are related

**Pull request merged**